src/main/java/uk/gov/hmcts/reform/sscs/service/HmcHearingApi.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import feign.codec.Decoder;
+import feign.codec.StringDecoder;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
```

Comment:
How do we feel about wildcard imports @dattipoeHMCTS @TeaWithLucas ?

Reply:
If there are 5+ imports from the same package it's fine. Good spot

Reply:
import org.springframework.*;  ?

Reply:
The checkstyle does not like wild imports...